src/AppBundle/Command/UpdateBundlesListCommand.php

```diff
@@ -108,6 +108,13 @@ private function getQuery()
      */
     private function getContentUpdateStruct(ContentService $contentService, $package)
     {
+        if ($package['stars'] == null) {
```

Comment:
For now, as a hot-fix it is fine, but I think that it should be handled in PackagistServiceProvider or even better in `knplabs/packagist-api`.

Reply:
Follow up: https://jira.ez.no/browse/COM-20029